### PR TITLE
Renaming verbose to verbosity in command line options

### LINKED ISSUE #187: rename long option from `verbose` to `verbosity`
for consistency with file-writer

will also need the system tests ini files updating

### PR DESCRIPTION
**Pull request by rerpha**
### Description of work

Breaking change: 

Kafka options are now the same as the filewriter and use `-S` or `--kafka-options`

Also it is now `--verbosity` rather than `--verbose` to set the verbosity level. 

### Issue

Closes #187 

### Acceptance Criteria

`MainOpt.cpp`
`KafkaW/BrokerSettings.cpp`

### Other
I have added an example of kafka options being specified in the `.ini` file in the `README.md`. 

---

## Code Review (To be filled in by the reviewer only)

- [ ] Is the code of an acceptable quality?
- [ ] Do the changes function as described and is it robust?

---

## Nominate for Group Code Review (Anyone can nominate it)
Indicate if you think the code should be reviewed in a Thursday code review session.

- [ ] Recommend for group code review

Also, nominate it on the code_review Slack channel (does someone want to automate this?).

### CODE REVIEW

**Comment by rerpha** [comment]
whilst i'm at it, i'm going to make the kafka-config options the same as the filewriter for topic options - using one option rather than a separate option for ints and strings.

**Comment by rerpha** [comment]
The system tests are hanging for some reason on this PR.

File: README.md [hunk 1]
```diff
@@ -37,6 +37,9 @@ with an `ini` file for command line options:
 broker = "//kafkabroker:9092"
 status-topic = "//kafkabroker:9092/the_status_topic"
 streams-json = ./streams.json
+kafka-config=consumer.timeout.ms 501 fetch.message.max.bytes 1234 api.version.request true
```
Comment (dominikwerder) [inline README.md]: Hm, isn't it way harder to read this INI style format compared to the former JSON, or is it only me?

Isn't it also harder for the machine? JSON maps easily in and out of Python, but this kind of configuration format needs explicit conversion logic.

File: src/MainOpt.cpp [hunk 2]
```diff
@@ -133,7 +133,7 @@ std::pair<int, std::unique_ptr<MainOpt>> parse_opt(int argc, char **argv) {
   App.add_option("--graylog-logger-address", opt.GraylogLoggerAddress,
                  "Address for Graylog logging");
   App.add_option("--influx-url", opt.InfluxURI, "Address for Influx logging");
-  App.add_option("-v,--verbose", log_level, "Syslog logging level", true)
+  App.add_option("-v,--verbosity", log_level, "Syslog logging level", true)
```
Comment (dominikwerder) [inline src/MainOpt.cpp]: Out of interest: What's the motivation for the change?

File: README.md (hunk 1 above)
Comment (SkyToGround) [reply README.md]: Yes it is only you. JSON is scourge upon humanity when used for application configuration (see [this](https://arp242.net/weblog/json_as_configuration_files-_please_dont), [this](https://www.lucidchart.com/techblog/2018/07/16/why-json-isnt-a-good-configuration-language/) and [this](https://revelry.co/json-configuration-file-format/) link), the ini files are almost always easier to read and the format allows comments!

File: src/MainOpt.cpp (hunk 2 above)
Comment (dominikwerder) [reply src/MainOpt.cpp]: Thanks! I just asked out of curiosity, I use anyway only the short version.

Comment (rerpha) [reply src/MainOpt.cpp]: me too, but for the INIs you have to put the long version in, i think.
